### PR TITLE
Add missing loglevel

### PR DESCRIPTION
**Pull request by erwinheitzman**
## Proposed changes

I believe that webdriver accepts the following logLevels: 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'silent'
Silent was missing from this list and it has been added in all the places needed.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mgrybyk** [comment]
Looks like your previous PR messed up with this one

File: packages/webdriverio/src/commands/element/selectByVisibleText.js [hunk 1]
```diff
@@ -38,14 +38,18 @@ export default async function selectByVisibleText (text) {
...
+
     /**
     * find option element using xpath
     */
-    const formatted = /"/.test(text)
-        ? 'concat("' + text.trim().split('"').join('", \'"\', "') + '")'
-        : `"${text.trim()}"`
-    const normalized = `[normalize-space(text()) = ${formatted}]`
-    const optionElement = await this.findElementFromElement(this.elementId, 'xpath', `./option${normalized}|./optgroup/option${normalized}`)
+    const formatted = /"/.test(normalized)
```
Comment (abjerstedt) [inline packages/webdriverio/src/commands/element/selectByVisibleText.js]: Can you back this out, and put this in your other selectByVisibleText PR?

File: packages/webdriverio/tests/commands/element/selectByVisibleText.test.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ describe('selectByVisibleText test', () => {
 
         expect(request.mock.calls[1][0].uri.path).toBe('/wd/hub/session/foobar-123/element')
         expect(request.mock.calls[2][0].uri.path).toBe('/wd/hub/session/foobar-123/element/some-elem-123/element')
-        expect(request.mock.calls[2][0].body.value).toBe('./option[normalize-space(text()) = "someValue1"]|./optgroup/option[normalize-space(text()) = "someValue1"]')
+        expect(request.mock.calls[2][0].body.value).toBe('./option[. = "someValue1"]|./optgroup/option[. = "someValue1"]')
```
Comment (abjerstedt) [inline packages/webdriverio/tests/commands/element/selectByVisibleText.test.js]: Same with these changes?

File: packages/webdriverio/src/commands/element/selectByVisibleText.js (hunk 1 above)
Comment (erwinheitzman) [reply packages/webdriverio/src/commands/element/selectByVisibleText.js]: Yup, will fix when I have the time. Strange that this got in.

**Comment by christian-bromann** [comment]
@abjerstedt are your concerns resolved?

**Pull request closed without merging**